IQSS/dataverse-frontend
# 295 - Display real dataset version information in the File Page

## What this PR does / why we need it:
This PR adds the integration with js-dataverse to get the dataset version information from the getFile use case to display the data in the SPA File Page

## Which issue(s) this PR closes:

- Closes #295 

## Special notes for your reviewer:

## Suggestions on how to test this:
### Step 1: Run the development environment
1. Run `npm I`
3. `cd packages/design-system && npm run build`
4. `cd ../../`
5. Check that you have a  `.env` file such as the .env.example, with the `VITE_DATAVERSE_BACKEND_URL=http://localhost:8000` variable
6.  `cd dev-env`
7. `./run-env.sh unstable`
8. To check the environment go to <http://localhost:8000> and check your local dataverse installation

### Step 2: Test breadcrumbs
1. Go to <http://localhost:8000>
2. Login as admin using username: dataverseAdmin and password: admin1
4. Create a new Dataset
5. Upload a file to the dataset
7. Go to <http://localhost:8000/spa> and select your dataset in the list
8. From the list of files in the Dataset page select your file to go to the File page
9. In the File Page check that the dataset version information such as the version labels and dataset citation matches the real dataset version information

## Does this PR introduce a user interface change? If mockups are available, please link/include them here:

## Is there a release notes update needed for this change?:

## Additional documentation:
